### DISCUSSION TITLE
Password verification field for users who change their own password

### DISCUSSION THREAD

**Discussion by temptuomas** [directus/directus#12204] [Feature Requests]
It would be nice if the user would be prompted for their old password if they are changing their own password. This could have security implications as well if an attacker could access to the user's session through cross-site scripting or cross-site request forgery and would be able to change the current user's password.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
